docs/guides/server/containers.adoc

```diff
@@ -128,6 +128,11 @@ To build the actual container image, run the following command from the director
 podman|docker build . -t mykeycloak
 ----
 
+[NOTE]
+====
+Podman can be used only for creating or customizing images. Podman is not supported in production environments.
+====
```

Comment:
Should this be a note for downstream only? In upstream we certainly support Podman and Docker in production (if I recall our previous discussion about it).

CC @ahus1

Reply:
Agreed. See below for suggestion that also clarifies the statement a bit with "Podman is not supported for running {project_name} in production environments."

Reply:
Looks good.